tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -69,10 +69,10 @@ public void testDescribeWithKRaft(ClusterInstance cluster) {
 
         // Change expected message to reflect latest MetadataVersion (SupportedMaxVersion increases when adding a new version)
         assertEquals("Feature: metadata.version\tSupportedMinVersion: 3.0-IV1\t" +
-                "SupportedMaxVersion: 4.0-IV0\tFinalizedVersionLevel: 3.3-IV1\t", outputWithoutEpoch(features.get(1)));
+                "SupportedMaxVersion: 3.8-IV1\tFinalizedVersionLevel: 3.3-IV1\t", outputWithoutEpoch(features.get(1)));
```

Comment:
I haven't had the time to fully check why this had to change from 4.0-IV0 to 3.8-IV1, but I will circle back tomorrow to confirm should you think it is a problem

Reply:
The test seems to fail with this change. This test is configured with unstable.feature.versions.enable=true. So, returning 4.0-IV0 for max supported version is correct.

Reply:
Yeah, upon rebasing this does need to be 4.0-IV0, I suspect my workspace was confused when I ran the test locally. Reverted!